types/products.d.ts

```diff
@@ -0,0 +1,19 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+declare const REMOTE_MODULES: Record<string, string>;
+
+declare module 'focalboard' {
```

Comment:
We need to provide TS declarations for the packages coming from modules since they don't exist in node_modules